backend/src/personnel/personnel.service.ts

```diff
@@ -219,13 +233,23 @@ export class PersonnelService {
     qb.leftJoinAndSelect('bcws_personnel.roles', 'roles');
     qb.leftJoinAndSelect('roles.role', 'role');
     qb.leftJoinAndSelect('bcws_personnel.homeFireCentre', 'homeFireCentre');
+    qb.leftJoinAndSelect('bcws_personnel.division', 'division');
 
-    this.addQueryBuilderCommonFilters(qb, query.name, query.availabilityType, query.availabilityFromDate, query.availabilityToDate);
+    this.addQueryBuilderCommonFilters(
+      qb,
+      query.name,
+      query.availabilityType,
+      query.availabilityFromDate,
+      query.availabilityToDate,
+    );
 
     if (query.fireCentre?.length) {
-      qb.andWhere('bcws_personnel.homeFireCentre.fireCentre IN (:...fireCentres)', {
-        fireCentres: query.fireCentre,
-      });
+      qb.andWhere(
+        'bcws_personnel.homeFireCentre.fireCentre IN (:...fireCentres)',
```

Comment:
```suggestion
        'homeFireCentre.fireCentre IN (:...fireCentres)',
```

Same change for line 256 please. That's on me